style.css

```diff
@@ -0,0 +1,102 @@
+#main-doc {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    background-color: #fff;
+    color: #000;
+    font-family: "Roboto", sans-serif;
+    font-size: 1.2em;
+    padding: 1em;
+    margin: 1em;
+  }
+  
+  #navbar {

```

Comment:
- [ ]  I can see you haven't done a change from the last review the code reviewer made. There's a 15 mins difference between your last change and her recent comment.
Kindly state the reason(s) for the suggested changes in a friendly manner, it may be obvious to you and not so to others. 👍

Reply:
It would be better to use the class selectors instead of the ID selectors as more elements could have this same style. This will make the code more readable.